### PR TITLE
fwupd: enable for ppc64 targets

### PR DESCRIPTION
**Pull request by q66**

### CODE REVIEW

File: srcpkgs/fwupd/template [hunk 1]
```diff
@@ -5,30 +5,39 @@ revision=1
 build_style=meson
 # manpages fail to build
 # tests require unpackaged umockdev
-configure_args="-Dbootdir=/boot/EFI -Dconsolekit=false -Dgtkdoc=false
- -Dintrospection=true -Dsystemd=false -Dplugin_altos=false -Dtests=false
- -Dman=false -Dpkcs7=false -Db_lto=false"
+configure_args="-Dconsolekit=false -Dgtkdoc=false -Dintrospection=true
+ -Dsystemd=false -Dplugin_altos=false -Dtests=false -Dman=false
+ -Dpkcs7=false -Db_lto=false"
+conf_files="/etc/dbus-1/system.d/org.freedesktop.fwupd.conf
```
Comment (maxice8) [inline srcpkgs/fwupd/template]: please keep it down there

**Comment by q66** [comment]
updated

File: srcpkgs/fwupd/template [hunk 2]
```diff
@@ -5,15 +5,14 @@ revision=1
 build_style=meson
 # manpages fail to build
 # tests require unpackaged umockdev
-configure_args="-Dbootdir=/boot/EFI -Dconsolekit=false -Dgtkdoc=false
```
Comment (maxice8) [inline srcpkgs/fwupd/template]: bootdir disappeared and never came back ?

Comment (q66) [reply srcpkgs/fwupd/template]: Yeah.Turns out it's not used by fwupd at all, was removed somewhere along the way. `find . -type f|xargs grep bootdir` shows nothing in their tree.

Comment (q66) [reply srcpkgs/fwupd/template]: All `-Dfoo` options also have to be present in `meson_options.txt`, you can see there is none: https://github.com/hughsie/fwupd/blob/master/meson_options.txt It's the same thing in the release archive, I checked.

**Pull request merged**